create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusS3ClientTest.java

```diff
@@ -0,0 +1,98 @@
+package no.sikt.nva.scopus;
+
+import com.amazonaws.SdkClientException;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.ObjectListing;
+import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.s3.model.S3ObjectInputStream;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import nva.commons.core.Environment;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class ScopusS3ClientTest {
```

Comment:
Tested by handler?

Reply:
I prefer to test classes/objects separately.  Now I know my class is doing what I expect it to do. And further up in the code (Handler) I can mock the whole thing.

Reply:
Ok. We will follow your premisses for testing.